implementations/rust/ockam/ockam_api/src/nodes/service/portals.rs

```diff
@@ -12,7 +12,7 @@ use ockam_abac::{Action, Expr, Resource, ResourceType};
 use ockam_core::api::{Error, Reply, Request, RequestHeader, Response};
 use ockam_core::errcode::{Kind, Origin};
 use ockam_core::{async_trait, route, AsyncTryClone, Route};
-use ockam_multiaddr::proto::Project;
+use ockam_multiaddr::proto::Project as ProjectProto;
```

Comment:
We could do that rename entirely. So many things called just `Project`.

Reply:
Then I'll need to rename all the other protos... Let's either not do it now or create a separate PR